src/libraries/Microsoft.Extensions.Configuration.Binder/src/Microsoft.Extensions.Configuration.Binder.csproj

```diff
@@ -28,4 +28,15 @@
   <ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
     <Compile Include="$(CommonPath)Extensions\ParameterDefaultValue\ParameterDefaultValue.netcoreapp.cs" Link="Common\src\Extensions\ParameterDefaultValue\ParameterDefaultValue.netcoreapp.cs" />
   </ItemGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\gen\Microsoft.Extensions.Configuration.Binder.SourceGeneration.csproj"
+                      ReferenceOutputAssembly="false"
+                      PackAsAnalyzer="true" />
+  </ItemGroup>
+
+  <PropertyGroup>
+    <IncludeMultiTargetRoslynComponentTargets>false</IncludeMultiTargetRoslynComponentTargets>
+    <RoslynComponentsAreOffByDefault>true</RoslynComponentsAreOffByDefault>
+  </PropertyGroup>
```

Comment:
Nit: move these properties to the top where the other properties reside. We usually segregate our project files in these three parts:
1. Properties
2. Compile items and other item types
3. Reference items (ProjectReferences, PackageReferences, ...)